wheel/python/clvm_rs/casts.py

```diff
@@ -0,0 +1,170 @@
+"""
+Some utilities to cast python types to and from clvm.
+"""
+
+from typing import Callable, List, SupportsBytes, Tuple, Union, cast
+
+from .clvm_storage import CLVMStorage, is_clvm_storage
+
+
+AtomCastableType = Union[
+    bytes,
+    str,
+    int,
+    SupportsBytes,
+]
+
+
+CastableType = Union[
+    AtomCastableType,
+    List["CastableType"],
+    Tuple["CastableType", "CastableType"],
+    CLVMStorage,
+]
+
+
+NULL_BLOB = b""
+
+
+def generate_working_memview_or_bytes():
+    # in python3.7 you can't go 'isinstance(b"", SupportsBytes)'
+
+    # this one works in py38+
+    def memview_or_bytes_py38_or_later(o):
+        return isinstance(o, (memoryview, SupportsBytes))
+
+    # this one works in py37
+    def memview_or_bytes_py37(o):
+        return getattr(o, "__bytes__", None) is not None
+
+    try:
+        memview_or_bytes_py38_or_later(b"")
+        return memview_or_bytes_py38_or_later
+    except TypeError:
+        pass
+    return memview_or_bytes_py37
+
+
+memview_or_bytes = generate_working_memview_or_bytes()
+
+
+def int_from_bytes(blob):
+    """
+    Convert a bytes blob encoded as a clvm int to a python int.
+    """
+    size = len(blob)
+    if size == 0:
+        return 0
+    return int.from_bytes(blob, "big", signed=True)
+
+
+def int_to_bytes(v) -> bytes:
```

Comment:
I can't find unit tests for this function, and `v` is missing type annotation

Reply:
Also in `test_program.py:test_int_round_trip` via `Program.int_to_bytes`.